src/gameLoader/gameLoader.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {Board} from "../board/board"
+import chalk from "chalk";
+import {getFilePath} from "../cli/cli";
+
+const CANNOT_READ_FILE_ERROR = "Can't read file; please check your -f argument point to a valid file.";
+
+export const loadGameConfigFromFile = ():Board => {
+    const filePath = getFilePath();
+    return parseBoardFromPath(filePath);
+}
+
+const parseBoardFromPath = (filePath:string):Board => {
+    let fileContent;
+    let board;
+
+    try{
+        fileContent = fs.readFileSync(filePath, 'utf8')
+    }
+    catch(error){
+        console.error(chalk.red(CANNOT_READ_FILE_ERROR));
+        return null;
+    }
+
+
+    try{
+        board = JSON.parse(fileContent);
+    }
+    catch(error){
+        console.error(chalk.red(CANNOT_READ_FILE_ERROR))
```

Comment:
Well, i think you should throw 2 different errors - one for file not found, and one for file not properly formatted. Otherwise, the message isn't informative enough.